datasets/health/query_test.json

```diff
@@ -0,0 +1,68 @@
+{
```

Comment:
we only need one health dataset. Maybe we can select the file with 15MB as the main one.